### DISCUSSION TITLE
Configuring TCP Binding by Network Interface Name

### DISCUSSION THREAD

**Discussion by gmaor88** [baresip/baresip#2680] [Q&A]
Hello,

I am utilizing Baresip to facilitate audio SIP calls on a device equipped with multiple network interfaces.
Through the utilization of the "-n" parameter, I'm directing the RTP stream output to a desired interface.

However, I am currently facing an issue with TCP communication binding.
As it stands, all TCP communication is routed through the Default Gateway.
I am eager to ascertain whether there exists a mechanism within Baresip that allows for TCP communication to be bound to a particular network interface by its name, rather than eztructing and using it's IP address.

Any insights or guidance you can provide regarding this matter would be greatly appreciated.

Thank you for your time and assistance.

Best regards

**Comment by sreimers** [comment]
`net_interface` config option should work https://github.com/baresip/baresip/wiki/Configuration#net_interface

**Comment by gmaor88** [comment]
Thank you for respnding sreimers,
Currently, I'm working with an older version, 0.6.4, due to legacy constraints.
I am already making use of the "-n" parameter to set the net_interface, but I see that all TCP communication is being directed through the default gateway.
Despite scanning the git log, I haven't been able to locate any changes that specifically address this particular issue.

**Comment by sreimers** [comment]
Since v0.6.4 we had many network related changes, we support only the last stable release and main.

**Reply by gmaor88** [reply]
I understand...
Thanks sreimers
